azure/Guidelines.md

```diff
@@ -374,6 +375,10 @@ Services, and the clients that access them, may be written in multiple languages
 
 :white_check_mark: **DO** use [RFC3339](https://datatracker.ietf.org/doc/html/rfc3339) for date/time.
 
+:white_check_mark: **DO** use a fixed time interval to express durations and include the time unit in the property name e.g., `backupTimeInMinutes` or `ttlSeconds`.
+
+:heavy_check_mark: **YOU MAY** use [RFC3339](https://datatracker.ietf.org/doc/html/rfc3339) durations only when users must be able to specify a time interval that may change from month to month or year to year e.g., "P1Y" represents 366 days on a leap year.
```

Comment:
Is that the right spec or did we mean ISO 8601?  The `Pwhatever` duration format is only discussed in the ABNF in the appendix and might not be as helpful as section 4.4.3 of ISO 8601?

Reply:
RFC 3339 supersedes ISO 8601, and the latter isn't accessible (paywalled) anywhere I can find outside of Wikipedia. So I don't think we can link to that, though I agree RFC 3339's ABNF isn't great. Should I link to Wikipedia as well / in lieu of?

Reply:
Yeah, I'd suggest linking to Wikipedia as well.  I don't think most internal non-HTTP-experts would be successful with just RFC 3339.